app/controllers/questions.js

```diff
@@ -1,55 +1,81 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    Question = mongoose.model('Question'),
-    _ = require('underscore');
+const mongoose = require('mongoose'),
+  Question = mongoose.model('Question');
+let region;
 
 
 /**
  * Find question by id
+ * @param {object} req
+ * @param {object} res
+ * @param {function} next
+ * @param {string} id
+ * @return {void}
+ *
  */
-exports.question = function(req, res, next, id) {
-    Question.load(id, function(err, question) {
-        if (err) return next(err);
-        if (!question) return next(new Error('Failed to load question ' + id));
-        req.question = question;
-        next();
-    });
+exports.question = (req, res, next, id) => {
+  Question.load(id, (err, question) => {
+    if (err) return next(err);
+    if (!question) return next(new Error(`Failed to load question ${id}`));
+    req.question = question;
+    next();
+  });
 };
 
 /**
  * Show an question
+ * @param {object} req
+ * @param {object} res
+ * @return {void}
  */
-exports.show = function(req, res) {
-    res.jsonp(req.question);
+exports.show = (req, res) => {
+  res.jsonp(req.question);
 };
 
 /**
  * List of Questions
+ *  @param {object} req
+  * @param {object} res
+  *  @return {void}
  */
-exports.all = function(req, res) {
-    Question.find({official:true, numAnswers: {$lt : 3}}).select('-_id').exec(function(err, questions) {
-        if (err) {
-            res.render('error', {
-                status: 500
-            });
-        } else {
-            res.jsonp(questions);
-        }
-    });
+exports.all = (req, res) => {
+  Question.find({ official: true, numAnswers: { $lt: 3 } }).select('-_id').exec((err, questions) => {
+    if (err) {
+      res.render('error', {
+        status: 500
+      });
+    } else {
+      res.jsonp(questions);
+    }
+  });
 };
 
+exports.setRegion = (regionId) => {
```

Comment:
this method is wrong, this would keep changing the region depending on which game sets it last